src/main/java/uk/ac/ebi/biostudies/service/file/chain/SendFileFilter.java

```diff
@@ -0,0 +1,338 @@
+package uk.ac.ebi.biostudies.service.file.chain;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import uk.ac.ebi.biostudies.service.file.FileMetaData;
+import uk.ac.ebi.biostudies.service.impl.FileDownloadServiceImpl;
+
+import javax.servlet.ServletOutputStream;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class SendFileFilter implements FileChainFilter{
```

Comment:
This should probably be a service/util method since it doesn't seem to be filtering anything.